cmd/openshift-tests/openshift-tests.go

```diff
@@ -286,4 +288,5 @@ func bindOptions(opt *testginkgo.Options, flags *pflag.FlagSet) {
 	flags.DurationVar(&opt.Timeout, "timeout", opt.Timeout, "Set the maximum time a test can run before being aborted. This is read from the suite by default, but will be 10 minutes otherwise.")
 	flags.BoolVar(&opt.IncludeSuccessOutput, "include-success", opt.IncludeSuccessOutput, "Print output from successful tests.")
 	flags.IntVar(&opt.Parallelism, "max-parallel-tests", opt.Parallelism, "Maximum number of tests running in parallel. 0 defaults to test suite recommended value, which is different in each suite.")
+	flags.StringVar(&opt.Report, "report", opt.Report, "Print the requested report at the end of the test suite.")
```

Comment:
Reading this further, is there any reason *not* to print the storage report on every storage test execution?  We definitely don't want to print it on dryrun (since dry run is specifically used to output test names to stdout, we don't want to print something else to stdout).  Otherwise we can simply print the storage report if we're not a dryrun and the suite is one of the csi suites.  That might look like:

```
if !opt.DryRun && len(args) > 0 && args[0] == "openshift/csi/..." {
  printStorageReport(...)
}
```

which removes the need for the flag.